src/screens/Settings/constants.ts

```diff
@@ -0,0 +1,57 @@
+type SettingInputs = Record<
+  string,
+  {
+    placeholder: string;
+    type: string;
+    isTextArea?: boolean;
+    validations?: {
+      [key: string]: {
+        value: string | number | boolean | RegExp;
+        message: string;
+      };
+    };
+  }
+>;
+
+export const WHITE = '#FFF';
+
+export const INPUTS: SettingInputs = {
+  image: {
+    placeholder: 'URL of profile picture',
```

Comment:
Be careful here, as this would always result in an english language placeholder, go with the i18n key here and use it in the component with the `t(key: string)` function.